cx/slack/server.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * TODO(developer): 
+ * Add your service key to the current folder.
+ * Uncomment and fill in these variables.
+ */
+// const slackSigningSecret = '...';
+// const slackToken = 'xoxb-...';
+// const projectId = 'my-project';
+// const locationId = 'global';
+// const agentId = 'my-agent';
+// const languageCode = 'en';
+
+const { WebClient } = require('@slack/web-api');
+const { createEventAdapter } = require('@slack/events-api');
+
+const slackEvents = createEventAdapter(slackSigningSecret);
+const slackClient = new WebClient(slackToken);
+
+// Imports the Google Cloud Some API library
+const {SessionsClient} = require('@google-cloud/dialogflow-cx');
+/**
+ * Example for regional endpoint:
+ *   const location = 'us-central1'
+ *   const client = new SessionsClient({apiEndpoint: 'us-central1-dialogflow.googleapis.com'})
+ */
+const client = new SessionsClient();
```

Comment:
How about auto-generate the regional Dialogflow endpoint from `locationId`?

Reply:
Changed the SessionClient to now auto-generate the regional Dialogflow endpoint from locationId.